depin/services/roles/xai/defaults/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+xai_cmd: "{{ depin_cmd | default('install') }}"
+xai_version: 1.1.14
+xai_metrics_port: 33004
+xai_download_url: "https://github.com/xai-foundation/sentry/releases/latest/download/sentry-node-cli-linux.zip"
```

Comment:
With the `depin.core.installer` the following should be updated.
```yml
xai_version: 1.1.14
xai_download_urls: 
  - "https://github.com/xai-foundation/sentry/releases/latest/download/sentry-node-cli-linux.zip"
```

The following should be moved to `depin/core/roles/metrics/default.yml`
```yml
xai_metrics_port: 33004
```